DuckDuckGo/TabViewController.swift

```diff
@@ -2416,18 +2378,6 @@ extension TabViewController: SecureVaultManagerDelegate {
                 }
             }
 
-            if !autoSavedCredentialsId.isEmpty {
```

Comment:
I think this var `autoSavedCredentialsId` can be safely deleted now that all that logic has moved to BSK